hasura/migrations/default/1705367428341_create_edit_rows_for_verified_apps/up.sql

```diff
@@ -0,0 +1,11 @@
+INSERT INTO
```

Comment:
do we need this backfilling? would be better if the frontend already handles this by letting users add an unverified row to make changes. bc when an app_metadata record moves from is_awaiting_review to verified by a reviewer, this row won't get created anyways

Reply:
Ok the front end currently handles it expecting the row to be there at all times and when approved by the reviewer they create another unverified row

Reply:
App rows without an edit row are supposed to be an edge case